src/Core/src/Platform/Windows/TransformationExtensions.cs

```diff
@@ -51,8 +51,6 @@ public static void UpdateTransformation(this FrameworkElement frameworkElement,
 				{
 					frameworkElement.RenderTransform = new CompositeTransform
 					{
-						CenterX = anchorX,
-						CenterY = anchorY,
```

Comment:
What happens if you do want the offset? Like a rotation about a different point on the element?

Reply:
That's already set in the above code on line 21:
`frameworkElement.RenderTransformOrigin = new global::Windows.Foundation.Point(anchorX, anchorY);`